components/leave_team_modal/leave_team_modal.tsx

```diff
@@ -40,11 +46,48 @@ export default class LeaveTeamModal extends React.PureComponent<Props> {
 
     handleSubmit = () => {
         this.props.onHide();
-        this.props.actions.leaveTeam(this.props.currentTeamId, this.props.currentUserId);
+        this.props.actions.leaveTeam(
+            this.props.currentTeamId,
+            this.props.currentUserId,
+        );
         this.props.actions.toggleSideBarRightMenu();
     };
 
     render() {
+        const numOfPublicChannels = this.props.publicChannels.length;
+        const numOfPrivateChannels = this.props.privateChannels.length;
+        const zero = 0;
+        let modalMessage;
+        if (numOfPublicChannels !== 0 && numOfPrivateChannels !== 0) {
```

Comment:
defining a const for zero seemed like a little odd, but if we want to go ahead you might as well replace the other instances here

Reply:
@sudheerDev  I have removed the constant for now keeping the conditions check to zero. If you want to change do let me know. I have also corrected the i18n error by running the script.